app/src/main/java/com/example/tasky/agenda_details/data/model/EventDetails.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.tasky.agenda_details.data.model
+
+data class EventDetails(
+    val id: String,
+    val title: String,
+    val description: String,
+    val from: Long,
+    val to: Long,
+    val remindAt: Long,
+    val attendeeIds: List<String>
+)
+
+sealed class PhotoType {
+    data class LocalPhoto(
+        val byteArray: ByteArray,
+        val uri: String
+    ) : PhotoType()
+
+    data class RemotePhoto(val key: String, val url: String) : PhotoType()
+}
```

Comment:
Looks great 👍 A few suggestions:
1. The class does not just represent a photo type, but a complete photo with everything that belongs to that, so could a name like `Photo` be more appropriate since it contains more than just the type information?
2. If you decide to use `Uri.parse(...)` for both local Uri and URL, you can make the Uri string a shared parameter by putting it into the constructor of the sealed class.